scripts/run_server.py

```diff
@@ -1,23 +1,26 @@
 import logging
 import argparse
 
-import numpy as np
-
-import vivarium.simulator.behaviors as behaviors
-from vivarium.controllers.config import SimulatorConfig, AgentConfig, ObjectConfig
-from vivarium.simulator.sim_computation import StateType
+from vivarium.simulator import behaviors
+from vivarium.simulator.states import init_simulator_state
+from vivarium.simulator.states import init_agent_state
+from vivarium.simulator.states import init_object_state
+from vivarium.simulator.states import init_nve_state
+from vivarium.simulator.states import init_state
 from vivarium.simulator.simulator import Simulator
 from vivarium.simulator.sim_computation import dynamics_rigid
-from vivarium.controllers.converters import set_state_from_config_dict
 from vivarium.simulator.grpc_server.simulator_server import serve
 
 lg = logging.getLogger(__name__)
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Simulator Configuration')
+    parser.add_argument('--custom_pos', action='store_true', help='Just a test arg to wether use custom or random pos')
     parser.add_argument('--box_size', type=float, default=100.0, help='Size of the simulation box')
     parser.add_argument('--n_agents', type=int, default=10, help='Number of agents')
+    parser.add_argument('--n_existing_agents', type=int, default=10, help='Number of agents')
     parser.add_argument('--n_objects', type=int, default=2, help='Number of objects')
+    parser.add_argument('--n_existing_objects', type=int, default=2, help='Number of agents')
```

Comment:
I agree with your proposition that " it would make more sense to rename the current n_agents to max_agents, and n_existing_agents to n_agents (same for objects)."

Reply:
Not sure if you already do it, but it will be useful to add an `assert` checking that the number of existing agents/objects does not exceed the max number

Reply:
Yes I already check this